includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -504,6 +504,7 @@ protected function layered_nav_list( $terms, $taxonomy, $query_type ) {
 			}
 
 			if ( ! empty( $current_filter ) ) {
+                asort($current_filter);
```

Comment:
Not following the WordPress coding standards. And using indentation as spaces.
Please correct the indentation and add spaces after `(` and before `)`.